driver/csiplugin/gpfs_util.go

```diff
@@ -260,17 +267,17 @@ func getScaleVolumeOptions(volOptions map[string]string) (*scaleVolume, error) {
 		scaleVol.IsFilesetBased = false
 	}
 
-	if fsTypeSpecified && isSCTypeSpecified {
-		return &scaleVolume{}, status.Error(codes.InvalidArgument, "The parameters \"type\" and \"filesetType\" are mutually exclusive")
+	if isSCAdvanced && fsTypeSpecified {
+		return &scaleVolume{}, status.Error(codes.InvalidArgument, "The parameter \"filesetType\" is not allowed with \"version\" value \""+storageClassAdvanced+"\"")
```

Comment:
Error message similar to previous error messages can be used here like ""parentFileset and filesetType=independent(Default) must not be specified together in storageClass")"

Reply:
Done